src/io/flutter/actions/FlutterSdkAction.java

```diff
@@ -120,5 +82,13 @@ public void actionPerformed(AnActionEvent event) {
     }
   }
 
-  public abstract void perform(@NotNull FlutterSdk sdk, @NotNull Project project, AnActionEvent event) throws ExecutionException;
+  public final void perform(@NotNull FlutterSdk sdk, @NotNull Project project, @Nullable AnActionEvent event, boolean logAction)
+    throws ExecutionException {
+    if (logAction) {
+      sendActionEvent();
+    }
+    perform(sdk, project, event);
```

Comment:
I generally put the implementation in the version of the method w/ the most args, but don't think we have any specific guidelines for such -